src/eclipse/eclipse.importorder

```diff
@@ -0,0 +1,7 @@
+#Organize Import Order
+#Wed Apr 26 12:53:22 EDT 2017
+4=\#
+3=org.springframework
+2=
+1=javax
+0=java
```

Comment:
Thank you for your feedback, @joaoandremartins !

So, look. My IDEA's config for import order is like:

![image](https://user-images.githubusercontent.com/1071590/26981233-76552004-4d02-11e7-94f3-6999c7f25004.png)

I guess your one is a bit different and this `eclipse.importorder` just does not have affect.

(I need that order in other projects)